src/__tests__/EventStore.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env jest */
```

Comment:
I don't think it is necessary to use the `.spec.js` or `.test.js` extension, since the tests are already under the `__tests__` directory so [the regex that jest uses by default to look for tests](http://jestjs.io/docs/en/configuration#testregex-string) will find them without issues. However I can understand why someone might still use different extensions so if you really want it I can rename it.

Reply:
That's true. I think it's fine to keep it as is. 👍